### DISCUSSION TITLE
Database sizing quetions

### DISCUSSION THREAD

**Discussion by STIAANWOL** [supabase/supabase#16876] [Questions] [answered]
Hi All!

This might be newbie questions but I need some clarity on how the sizing is calculated on the database.

I have 2500 records with an embeddings column which is 5% of the size of my properties table.
The properties table is 270mb according to screenshot. But the rest of the table is extremely minimal data points like integers of amount of bedrooms etc.

And the total size of my db is 380mb.

Im wondering if these numbers sounds correct and if theres maybe additional storage thats happening under the hood?

My 'Average Database size per day' is also 1.3GB. Its not matching up for me.

Some clarity would be appreciated!

![sb1](https://github.com/supabase/supabase/assets/8208674/1e7056c3-51d6-490c-9508-3c17ccabb054)
![sb4](https://github.com/supabase/supabase/assets/8208674/57d5df88-1679-4534-92ee-a92578768774)
![sb2 PNG](https://github.com/supabase/supabase/assets/8208674/b0eaaa0c-0264-47fa-a93c-105cbb893ca4)
![sb3](https://github.com/supabase/supabase/assets/8208674/bdc6dd2d-fa80-407b-bfc1-3fb0bf8177e3)

**Comment by GaryAustin1** [comment, marked as answer]
Can't comment on the size of your table.  You can google on the web how postgres tables get stored and sizes.  It is very complicated to estimate the size a table will be.

An empty instance with no tables can have a database size of 70 to 90Mb.  That includes all of postgres tables and extensions, as well as all the supabase auth, vault, realtime, and storage tables.

**Reply by STIAANWOL** [reply]
Thank you for your reply! That helped. I appreciate your time.

**Comment by STIAANWOL** [comment]
For those coming after me...
I was referred to this by Supabase support https://supabase.com/docs/guides/platform/database-size#database-size
in which its mentioned that 'New Supabase projects have a database size of ~40-60mb.' here https://supabase.com/docs/guides/platform/database-size#preoccupied-space

**Reply by STIAANWOL** [reply]
Its also worth noting that on the Pro plan you get 8GB and then its $0.125 per GB there after
![image](https://github.com/supabase/supabase/assets/8208674/b345aec5-e1a8-4e37-8a27-0205d23ccdd2)
